<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileReaderNodeDialog</class>
 <widget class="QWidget" name="FileReaderNodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>275</width>
    <height>355</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File Reader Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_structureLabel">
     <property name="text">
      <string>File Structure:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Spire::FileReaderNodeStructureTable" name="m_structureTable">
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_errorHandlingLayout">
     <item>
      <widget class="QLabel" name="m_errorHandlingLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Error Handling:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_errorHandlingComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Spire::FileReaderNodeStructureTable</class>
   <extends>QTableWidget</extends>
   <header>Spire/CanvasView/FileReaderNodeStructureTable.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
